homework9/src/components/SiteNav/SiteNav.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+import "./SiteNav.css";
+
+let activeStyle = {
+  color: "#59abb0",
+};
+
+function SiteNav() {
+  return (
+    <ul className="list site-nav">
+      <li>
+        <NavLink
```

Comment:
for that navigation you can create array with object that include path, title and some other keys(if need it) 
Otherwise for every `li` you repeat yourself